packages/steppp/src/index.ts

```diff
@@ -87,6 +88,34 @@ function Steppp(element: HTMLElement, options: any = defaultOptions): Instance {
     ];
   };
 
+  /**
+   * While animating, we need to "freeze" the dimensions of each step, since they'll be
+   * absolutely positioned and removed from the document flow. Then, we can reset.
+   */
+  const transitionDimensions = (oldStep: HTMLElement, newStep: HTMLElement) => {
```

Comment:
Only during animations, we need to take a height/width "snapshot" of the target steps, so that the animation can flow without the steps breaking the flow. After animation is complete, we can then safely remove the rules and allow it to render normally again.